src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -263,6 +271,8 @@ public SipGatewaySession(SipGateway gateway, CallContext callContext)
             = sipProvider.getOperationSet(
                     OperationSetJitsiMeetTools.class);
 
+        jitsiMeetTools.addRequestListener(this);
```

Comment:
Hum seems we are already setting this listener

Reply:
Why do you need it here?

Reply:
We should either remove the other one and make sure we clean up, as this will leak sessions in memory.